apps/src/templates/projects/projectsRedux.js

```diff
@@ -145,6 +148,32 @@ function personalProjectsList(state = initialPersonalProjectsList, action) {
         ...state,
         projects: projects,
       };
+    case START_RENAMING_PROJECT:
+      var projectBeingRenamed = action.projectId;
+
+      var projectBeingRenamedIndex = state.projects.findIndex(project => project.channel = projectBeingRenamed);
```

Comment:
Here and below: You've got a single-equals assigning `projectBeingRenamed` to `project.channel`.

Reply:
Aha! Good catch. I only had one project so I didn't notice. Thank you!